controllers/tasksController.js

```diff
@@ -27,7 +27,7 @@ const addNewTask = async (req, res) => {
  */
 const fetchTasks = async (req, res) => {
   try {
-    const allTasks = await tasks.fetchTasks()
+    const allTasks = await tasks.fetchTasks(req.body)
```

Comment:
Any specific reason of passing `req.body` here?

Reply:
no actually, I think I added it previously when I added the id-mapping logic in this file. But the code's working fine otherwise. Should I remove it?

Reply:
If we don't need it, we should remove it

Reply:
yeah ok. I'll remove it then